network-api/networkapi/static/README.md

```diff
@@ -0,0 +1 @@
+why is this directory required?
```

Comment:
LOL

Reply:
it won't work without this directory? Probably because we set the `STATICFILES_DIR` setting. If we remove that, we won't need this folder anymore.